src/Dialect/ONNX/ONNXOps/Math/STFT.cpp

```diff
@@ -0,0 +1,90 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+//===----------------- STFT.cpp - Lowering STFT Op
+//--------------------------===//
+//
+// Copyright 2019-2024 The IBM Research Authors.
+//
+// =============================================================================
+//
+// This file provides definition of ONNX dialect STFT operation.
+//
+//===----------------------------------------------------------------------===//
+
+#include "src/Dialect/ONNX/DialectBuilder.hpp"
+#include "src/Dialect/ONNX/ONNXOps/OpHelper.hpp"
+
+using namespace mlir;
+using namespace mlir::OpTrait::util;
+using namespace onnx_mlir;
+
+//===----------------------------------------------------------------------===//
+// Support
+//===----------------------------------------------------------------------===//
+
+namespace onnx_mlir {
+
+template <>
+LogicalResult ONNXGenericSTFTOpShapeHelper<ONNXSTFTOp>::computeShape() {
+  typename ONNXSTFTOp::Adaptor operandAdaptor(
```

Comment:
```suggestion
  ONNXSTFTOpAdaptor operandAdaptor(
```

I do not think the typename is needed here